build.gradle

```diff
@@ -193,14 +194,11 @@ task buildAwsLc {
         }
 
         if (!isLegacyBuild) {
-            exec {
-                workingDir awslcSrcPath
-                commandLine "git", "fetch", "--tags"
-            }
-            exec {
-                workingDir awslcSrcPath
-                commandLine "git", "checkout", awsLcGitVersionId
-            }
+            // FIXME: Commented because it was breaking Github CI
```

Comment:
What's going on here?

Reply:
Github does not support `git checkout <branch>` in submodules. This is temporary till aws-lc merges the HPKE PR, once that's done, I'll add this back and replace the VersionId with a commit hash.

Sorry for not explicitly marking these temporary changes.